src/com/jpexs/decompiler/flash/gui/MainPanel.java

```diff
@@ -424,7 +424,9 @@ private JPanel createWelcomePanel() {
     private JPanel createFolderPreviewCard() {
         JPanel folderPreviewCard = new JPanel(new BorderLayout());
         folderPreviewPanel = new FolderPreviewPanel(this, new ArrayList<>());
-        folderPreviewCard.add(new JScrollPane(folderPreviewPanel), BorderLayout.CENTER);
+	final JScrollPane jScrollPane = new JScrollPane(folderPreviewPanel);
```

Comment:
I don't understand why the intendation here is different, it looks fine on my local editors. Might be because I use tabs instead of spaces